pages/price-feeds/use-real-time-data/evm.mdx

```diff
@@ -70,70 +48,40 @@ import "@pythnetwork/pyth-sdk-solidity/PythStructs.sol";
 contract SomeContract {
   IPyth pyth;
 
+  /**
+   * @param pythContract The address of the Pyth contract
+   */
   constructor(address pythContract) {
+    // The IPyth interface from pyth-sdk-solidity provides the methods to interact with the Pyth contract.
+    // Instantiate it with the Pyth contract address from https://docs.pyth.network/price-feeds/contract-addresses/evm
     pyth = IPyth(pythContract);
   }
 
-  function doSomething(
-    uint someArg,
-    string memory otherArg,
-    bytes[] calldata priceUpdateData
-  ) public payable {
-    // Update the on-chain Pyth price(s)
-    uint fee = pyth.getUpdateFee(priceUpdateData);
-    pyth.updatePriceFeeds{ value: fee }(priceUpdateData);
-
-    // Read the current price
-    bytes32 priceId = 0xf9c0172ba10dfa4d19088d94f5bf61d3b54d5bd7483a322a982e1373ee8ea31b;
+  /**
+     * This method is an example of how to interact with the Pyth contract.
+     * Fetch the priceUpdate from Hermes and pass it to the Pyth contract to update the prices.
+     * See https://docs.pyth.network/price-feeds/fetch-price-updates for more information on how to fetch the priceUpdate.
+
+     * @param priceUpdate The encoded data to update the contract with the latest price
+     */
+  function exampleMethod(bytes[] calldata priceUpdate) public payable {
+    // Submits a priceUpdate to the Pyth contract to update the on-chain price.
```

Comment:
this should say "Submit" for consistency in verb tense with the rest of the comments